### DISCUSSION TITLE
Can't RadioButton have value other than string or undefined?

### DISCUSSION THREAD

**Discussion by hslee2008** [carbon-design-system/carbon-components-svelte#1866] [General]
Currently, carbon-components-svelte only allows type of `string | undefined` in value (RadioButton).

![image](https://github.com/carbon-design-system/carbon-components-svelte/assets/78584173/6d2b41de-4704-4088-ae4a-aeeb783bf74f)

But there are many cases where the value needs to be numbers.

I mean, putting in numbers does work but the types only say `string` is allowed.

**Comment by metonym** [comment]
No reason in particular. I think it's completely reasonable to loosen the type to allow numbers.

**Comment by metonym** [comment]
Fixed in [v0.82.7](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.82.7) by https://github.com/carbon-design-system/carbon-components-svelte/pull/1865
